lms/urls.py

```diff
@@ -143,6 +143,8 @@
 
     url(r'^dashboard/', include('learner_dashboard.urls')),
     url(r'^api/experiments/', include('experiments.urls', namespace='api_experiments')),
+
```

Comment:
We mostly use some kind of tracker (like [COLARAZ_CUSTOM] or [UCSD_CUSTOM]) with a comment, whenever we do any change in edX core. 
It will help us in the future to find what we have changed in the core code for upgrades.

Reply:
okay updating